### PR TITLE
LPS-153789 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-153789

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase [hunk 1]
```diff
@@ -278,6 +278,76 @@ definition {
...
+
+			SelectFrameTop();
+
+			SelectFrame(
+				key_title = "Add Site",
+				locator1 = "IFrame#MODAL_ANY");
+
+			Click(
+				key_text = "Done",
+				locator1 = "Modal#MODAL_FOOTER_BUTTON");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase:337]: AssetCategorization.selectCategories

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase [hunk 2]
```diff
@@ -278,6 +278,76 @@ definition {
...
+		}
+
+		task ("Then the site creator can successfully add the site") {
+			Button.clickAdd();
+
+			SelectFrameTop();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase:343]: Redundant step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase [hunk 3]
```diff
@@ -278,6 +278,76 @@ definition {
...
+
+			Alert.viewSuccessMessage();
+
+			ControlMenu.viewHeaderTitle(headerTitle = "Site Settings");
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase:348]: Add more step to tear down the global vocabulary

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase (hunk 1 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesUseCase.testcase:337]: Hi Tim! I had already used this macro but I didn't work. I also thought about adding a condition inside this macro but the logic would be weird.

When we select the category we must before select the IFrame vocabulary, the weird part is now. Although the Done and Cancel button seems to be at the same IFrame they are not, these buttons are inside IFrame Add Site then we have to select the Top and select the Add Site IFrame again.

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao! I've pushed a new commit and wrote a comment. Would you please have a loo at? Thanks!

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/pull/9311
@ozysouza

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**